tcms/rpc/tests/test_testexecution.py

```diff
@@ -37,6 +37,34 @@ def test_add_comment_with_pk_as_int(self):
         self.assertEqual(created_comment['comment'], first_comment.comment)
 
 
+class TestExecutionRemoveComment(APITestCase):
+    """Test TestExecution.remove_comment"""
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        self.execution = TestExecutionFactory()
+
+    def test_delete_all_comments_with_pk(self):
+        self.rpc_client.TestExecution.add_comment(self.execution.pk, "Hello World!")
+        self.rpc_client.TestExecution.add_comment(self.execution.pk, "More comments")
+        self.rpc_client.TestExecution.remove_comment(self.execution.pk)
+
+        comments = get_comments(self.execution)
+        self.assertEqual(comments.count(), 0)
+
+    def test_delete_one_comment_with_pk(self):
```

Comment:
Naming is generally good here but it doesn't differentiate between "removing all comments from a TE" and "removing a specific comment from TE specified by its PK".  The wording "_with_pk" is ambiguous because both API parameters are primary keys anyway.